specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -9393,7 +9393,7 @@
         },
         "hyperVGeneration": {
           "$ref": "#/definitions/HyperVGenerationType",
-          "description": "Gets the HyperVGenerationType of the VirtualMachine created from the image"
+          "description": "Specifies the HyperVGenerationType of the VirtualMachine created from the image. This propery might be required if not available from the blob or disk resource."
```

Comment:
We need to document that if the source is blob then we need the user to specify this field and in case the source is a managed resource like disk or snapshot the user may need to specify this property if we can;t deduce it from the source managed resource, and we should reference the disk and snapshot documentation reference where the user can check the hyperVGeneration property.